src/plugins/workspace/public/utils.ts

```diff
@@ -55,3 +55,18 @@ export const featureMatchesConfig = (featureConfigs: string[]) => ({
 
   return matched;
 };
+
+export const getSelectedFeatureQuantities = (
```

Comment:
Seems test cases for this function is missing.

Reply:
Test cases will be added.